### PR TITLE
Set up CI with Azure Pipelines

### LINKED ISSUE #159: Upgrade Dockerfile
debian stretch has been moved to archives. We should take the opportunity to upgrade the base file, i.e. switching to something like **rocker/r-ver:4.2.3**, which however is based on Ubuntu (22.04). While upgrading to a new Ubuntu is good, it will require some adaptations in the installations of pandoc and texlive, beyond some straight-forward library substitutions.

### PR DESCRIPTION
**Pull request by RolandASc**
addresses #159 

**app itself needs to be tested and fixed / updated for the new version of R and packages!**

maybe take up #109 during PR review?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,19 +31,24 @@ RUN apt-get update \
 # - https://pandoc.org/installing.html#linux
 # We should use the same version as in rocker/rstudio:<R_VER>
 #   docker run --rm rocker/rstudio:<R_VER> /usr/lib/rstudio-server/bin/pandoc/pandoc -v
+# RSc: TODO: need to check and update pandoc version here
```
Comment (RolandASc) [inline Dockerfile]:
```suggestion
#   docker run --rm rocker/rstudio:4.2.3 /usr/lib/rstudio-server/bin/quarto/bin/tools/pandoc -v
#   --> pandoc 2.19.2
```

**Comment by RolandASc** [comment]
I would propose to merge this as is and handle #109 later as a separate thing.

Upgrade of pandoc or potential migration to something quarto-based should probably also fit into a new separate issue and doesn't need to be resolved here.

**Comment by RolandASc** [comment]
@riccardoporreca , @GuidoMaggio , just stumbled over this one, I think this should have been merged last November (pending an approval)

**Comment by riccardoporreca** [comment]
@RolandASc, I have run manually a pipeline, since the last one was 1yo https://dev.azure.com/miraisolutions/MiraiApps/_build/results?buildId=1137

All good

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Docker
+# Build and push an image to Azure Container Registry
+# https://docs.microsoft.com/azure/devops/pipelines/languages/docker
+
+trigger:
+- develop
```
Comment (riccardoporreca) [inline azure-pipelines.yml:7]: We want to have this also on `master`, which is where we would usually deploy from, although I guess the the automated deployment from `master` should be re-established

```suggestion
- master
- develop
```

**Review by riccardoporreca — APPROVED** [review]

**Pull request merged**